AnnoyancesFilter/sections/self-promo.txt

```diff
@@ -214,6 +214,9 @@ analyticsvidhya.com##a[id^="stickyban"]
 testbook.com##.post-body-banner a[href="https://testbook.com/pass"] > img
 ! promotional ads
 kakakumag.com##.side-ad-box
+kakakumag.com###_popIn_recommend
```

Comment:
It seems all these 3 rules changes nothing

Reply:
Also note popIn is fully blocked only if TPF is enabled, so this kind of filter should go on Spyware > specific.

Reply:
> It seems all these 3 rules changes nothing

The reason is that `.container-side-wrap` has fixed height. I don't want to override this just to remove small placeholder at the bottom.

Reply:
ok. I delete these 3 rules

Reply:
@SKEIDs Without tracking protection filter the links in the popin widget lead to site's another articles - that's why we don't remove this widget in adblock filters.